packages/fuzzer/package.json

```diff
@@ -16,9 +16,10 @@
 	"main": "dist/fuzzer.js",
 	"types": "dist/fuzzer.d.ts",
 	"scripts": {
-		"prebuild": "prebuild --runtime napi --backend cmake-js --all --strip --verbose",
-		"install": "prebuild-install --runtime napi || npm run prebuild",
-		"build": "cmake-js build --out build",
+		"prepack": "prebuild --runtime napi --backend cmake-js --all --strip --verbose",
```

Comment:
`prepack` is executed during publishing of the npm projects. Is this really necessary? We probably can use a custom script name as it's explicitly called during install. WDYT?

Reply:
I'm unsure of how `prebuild` (the program) intends itself to be used but `prebuild` (the script) is also called before `build` which I don't think is what we wanted. I used `prepack` because it made sense to me that we would do this prebuilding only when publishing the package, though I could see using a different name to avoid these lifecycle issues completely.

Reply:
Yes, a different name would probably untangle this somewhat. 
The `prebuild` command is only used in the `prerelease` pipeline, uploading the artifacts to GitHub, and during `install`, if no matching artifact could be found online. Those should be all places that would need to change.